v0x04/match_fields.py

```diff
@@ -13,6 +13,30 @@
 from pyof.v0x04.common.flow_match import OxmOfbMatchField, OxmTLV, VlanId
 
 
+def mask_to_bytes(mask, size):
+    """Return the mask in bytes."""
+    bits = 0
+    for i in range(size-mask, size):
+        bits |= (1 << i)
+    tobytes = bits.to_bytes(size//8, 'big')
+    return tobytes
+
+
+def bytes_to_mask(tobytes, size):
+    """Return the mask in string."""
+    int_mask = int.from_bytes(tobytes, 'big')
+    bits = bin(int_mask)
+    strbits = str(bits)
+    strbits = strbits[2:]
+    netmask = 0
+    for i in range(size):
+        if strbits[i] == '1':
+            netmask += 1
+        else:
+            break
+    return netmask
+
+
```

Comment:
There is a `utils.py` module in `v0x04` directory. These auxiliary functions should be there.

Reply:
Hi, it seems like moving these functions to utils.py and import them from utils are causing the updated commit to fail the Scrutinizer check.